common/lib/xmodule/xmodule/services.py

```diff
@@ -0,0 +1,19 @@
+"""
+Module contains various XModule/XBlock services
+"""
+from django.conf import settings
+
+class SettingsService(object):
+    """
+    Allow XBlocks to access the server settings
+    """
+    xblock_settings_bucket_selector = 'block_settings_key'
+
+    def get_settings_bucket(self, block, default=None):
+        """ Gets xblock settings dictionary from settings. """
+        if not block:
+            raise ValueError("Expected XBlock instance, got {0}".format(block))
```

Comment:
Will this really give a useful error message? Sometimes I find just printing out the object itself is less informative than printing out some additional information, such as the type of the object